src/cbmc_starter_kit/template-for-repository/proofs/Makefile.common

```diff
@@ -303,6 +303,9 @@ COMPILE_FLAGS ?= -Wall
 LINK_FLAGS ?= -Wall
 EXPORT_FILE_LOCAL_SYMBOLS ?= --export-file-local-symbols
 
+# During instrumentation, it adds models of C library functions
+ADD_LIBRARY_FLAG ?= --add-library
```

Comment:
If this is supposed to be switched on all the time, please use `:=`.

If users should be able to switch it off, please do
```
  ifeq ($(strip $(NO_ADD_LIBRARY_FLAG)),)
    CBMC_ADD_LIBRARY_FLAG := --add-library
  else
    CBMC_ADD_LIBRARY_FLAG := 
  endif
```

Reply:
Implemented using `:=` as suggested.